resources/logback.xml

```diff
@@ -15,8 +15,8 @@
 
   <logger name="org.apache"      level="ERROR" />
   <logger name="org.symphonyoss" level="ERROR" />
-  <logger name="org.jolokia"     level="ERROR" />
-
+  <logger name="org.jolokia"     level="INFO" />
+  <logger name="org.symphonyoss.client.impl.SymphonyBasicClient" level="INFO" />
```

Comment:
Is this a hangover from development / debugging?  This should not be the default logging setting.

Reply:
Jolokia log level is, but the SJC is IMO useful, as it logs if/when the JMX MBean is exposed; I'll take it out if you don't like it.

Reply:
I've found SJC to be too noisy to be left at `INFO` level, though I realise you're only setting it on a single class.

Regardless, best practice is to set logging to `ERROR` only by default - it can always be overridden in any particular environment when needed (including via JMX!).